test/package.test.js

```diff
@@ -147,6 +147,16 @@ describe('package', function () {
         Expect(version).to.eql('1.2.3');
       });
     });
+
+    it('returns null if no version is specified', function () {
+      Package.getUserPackage.restore();
+      Sinon.stub(Package, 'getUserPackage').returns(Bluebird.resolve({ version: '' }));
+
+      return Package.calculateNewVersion()
+      .then(function (version) {
+        Expect(version).to.eql(null);
```

Comment:
Nit pick: can you change this to [`.to.be.null;`](http://chaijs.com/api/bdd/#method_null). We prefer to use the chai shorthands whenever possible.